documentation/templates/bolt_pwsh_reference.md.erb

```diff
@@ -0,0 +1,45 @@
+# Bolt PowerShell Command Reference
+
+The Bolt PowerShell module contains  PowerShell native cmdlets auto-generated from Puppet Bolt source code with correct PowerShell verb-noun semantics, familiar PowerShell parameter conventions, and identical Puppet Bolt help documentation.
```

Comment:
```suggestion
The Bolt PowerShell module contains PowerShell native cmdlets auto-generated from Bolt source code with correct PowerShell verb-noun semantics, familiar PowerShell parameter conventions, and identical Bolt help documentation.
```

Reply:
there were more instances of 'Puppet Bolt', so i fixed all instead of adding this commit suggestion